src/main/java/mate/academy/bookstorev2/service/UserServiceImpl.java

```diff
@@ -22,6 +31,25 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto) {
                     + " already exist");
         }
         User user = userMapper.toModel(requestDto);
+        user.setPassword(passwordEncoder.encode(requestDto.getPassword()));
+
+        Role role;
+        if (user.getEmail().contains("admin")) {
```

Comment:
why do you check it?

Reply:
I wanted to check the posibility of assigning roles. So I decided to do it that way.

Reply:
if I`ll had admin in my email it`s mean that I can be admin? not good idea